components/cdc/src/service.rs

```diff
@@ -414,18 +415,26 @@ mod tests {
             }
         };
 
-        let mut event_small = Event::default();
         let row_small = EventRow::default();
-        let mut event_entries = EventEntries::default();
-        event_entries.entries = vec![row_small].into();
-        event_small.event = Some(Event_oneof_event::Entries(event_entries));
+        let event_entries = EventEntries {
+            entries: vec![row_small].into(),
+            ..Default::default()
+        };
+        let event_small = Event {
+            event: Some(Event_oneof_event::Entries(event_entries)),
+            ..Default::default()
+        };
 
-        let mut event_big = Event::default();
         let mut row_big = EventRow::default();
-        row_big.set_key(vec![0 as u8; CDC_MAX_RESP_SIZE as usize]);
-        let mut event_entries = EventEntries::default();
-        event_entries.entries = vec![row_big].into();
-        event_big.event = Some(Event_oneof_event::Entries(event_entries));
+        row_big.set_key(vec![0_u8; CDC_MAX_RESP_SIZE as usize]);
```

Comment:
Why doesn't clippy complain about it 🤣

Reply:
What do you mean?

Reply:
I don't have a strong preference for the style. I just don't know why there isn't a warning about replacing it with:

```
EventRow {
   key: xxx,
   ..Default::default()
}
```

Reply:
I guess clippy won't look into `set_key`'s detail, or check its name for that matter. I'll leave it be for now.